operator/roles/csi-scale/templates/cluster-role-snapshotter.yaml.j2

```diff
@@ -10,14 +10,6 @@ metadata:
   name: ibm-spectrum-scale-csi-snapshotter
   namespace: "{{ namespace }}"
 rules:
-- apiGroups:
-  - ''
-  resources:
-  - secrets
```

Comment:
Is access to secrets not needed y snapshotter?

Reply:
Yes Smita, this secret access is optional and only required if CSI driver needs secret in in VolumeSnapshotClass.

More  details here :
https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.0/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml